src/SemesterPage.tsx

```diff
@@ -5,6 +5,10 @@ import { RootState } from "./duck";
 import { useSelector } from "react-redux";
 import React from "react";
 
+/*
```

Comment:
Again, is this a comment that actively improves our knowledge of the component? Or in other words, does this comment tell the reader more than reading the source code of the component?